### PR TITLE
Upgraded vis-util to 1.1.4

### PR DESCRIPTION
**Pull request by ilancohen**
This should fix various issues with the use of `deepExtend` and `selectiveDeepExtend`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,18 +40,19 @@
...
+    "vis-util": "~1.1.4"
   },
   "devDependencies": {
     "@babel/core": "^7.5.4",
     "@babel/preset-env": "^7.5.4",
     "@babel/register": "^7.4.4",
     "cpy-cli": "^2.0.0",
     "eslint": "^4.3.0",
+    "git-authors-cli": "^1.0.18",
+    "ignore-styles": "^5.0.1",
```
Comment (mojoaxel) [inline package.json]: where is `ignore-styles` used?

File: package.json [hunk 2]
```diff
@@ -40,18 +40,19 @@
     "moment": "^2.20.1",
     "propagating-hammerjs": "^1.4.6",
     "vis-data": "^6.1.1",
-    "vis-util": "^1.1.3"
+    "vis-util": "~1.1.4"
```
Comment (mojoaxel) [inline package.json]: Please change this back to a caret range using `^`. We would like to auto update to minor version changes.
See https://docs.npmjs.com/misc/semver#tilde-ranges-123-12-1

Comment (ilancohen) [reply package.json]: Done.

File: package.json (hunk 1 above)
Comment (ilancohen) [reply package.json]: Mistake. I've removed it.

**Comment by mojoaxel** [comment]
@ilancohen :tada: Thanks for your contribution!

**Pull request merged**